backend/lambdas/generators/json_report/json_report/report.py

```diff
@@ -39,12 +41,22 @@ def get_report(scan_id, params=None):
     if "results" not in params or "inventory" in params["results"]:
         inv_results = get_inventory(scan)
 
+    if "results" not in params or "configuration" in params["results"]:
+        config_results = get_configuration(scan)
+
     errors.update(vuln_results.errors)
     errors.update(secret_results.errors)
     errors.update(sa_results.errors)
     errors.update(inv_results.errors)
+    errors.update(config_results.errors)
 
-    success = vuln_results.success and secret_results.success and sa_results.success and inv_results.success
+    success = (
```

Comment:
this is one area where I would think that config plugin results may potentially differ from inventory plugins - that we may want failed config checks to contribute to the overall pass/fail `success` of the scan results. if any vuln or static analysis findings are identified (even if lower severity), it still accounts for a "failing" (`false`) `success`, unless a user is specifically filtering for certain criteria, e.g., `severity=critical&severity=high`. 

I would think we would want to replicate that with config results.. if there are no filters, then any test failure contributes to a failed overall result status. **this looks like what you are doing here in this code.**

we may want to discuss offline whether it makes sense to also provide an option to filter-out certain config checks from a failing status result.